doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -620,140 +620,90 @@ causes no error message, although (s1, s1) is probably a user error.
 Table Constraint Definition for foreign keys
```

Comment:
Maybe fix capitalization in this and previous heading while we are here?
<img width="237" alt="image" src="https://github.com/tarantool/doc/assets/42832629/a0ef451d-6c72-49d6-a03f-387fbf25f7b6">